Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.8 as base
+
+#install poetry
+RUN pip3 install poetry
+
+# copy across pyproject.toml and poetry.lock so the dependencies can be installed
+RUN mkdir /app
+COPY ./poetry.lock /app
+COPY ./pyproject.toml /app
+WORKDIR /app
+
+#install poetry dependencies from pyprojecy.toml
+RUN poetry install
```

Comment:
Again, minor but here we could take advantage of `poetry install --no-dev` which would ensure our production image skips installing our dev dependencies (though we would then need to change the dev stage to install those!)